kubert/src/server.rs

```diff
@@ -4,7 +4,9 @@
 //! Unlike a normal `hyper` server, this server reloads its TLS credentials for each connection to
 //! support certificate rotation.
 
-use std::{convert::Infallible, net::SocketAddr, path::PathBuf, str::FromStr, sync::Arc};
+use std::{
+    convert::Infallible, io::BufReader, net::SocketAddr, path::PathBuf, str::FromStr, sync::Arc,
```

Comment:
if we follow my other suggestions, i don't think we need the `BufReader` import?
```suggestion
    convert::Infallible, net::SocketAddr, path::PathBuf, str::FromStr, sync::Arc,
```